contracts/SpecificGroupStrategy.sol

```diff
@@ -127,12 +140,6 @@ contract SpecificGroupStrategy is UUPSOwnableUpgradeable, UsingRegistryUpgradeab
      */
     error NoActiveGroups();
 
-    /**
-     * @notice Used when attempting to deposit when chosen strategy cannot receive any new votes.
-     * @param group The group's address.
-     */
-    error StrategyCannotReceiveVote(address group);
```

Comment:
I was thinking about it in a way that we can always activate new validator group and rebalance. Since we are lifting group count limit there should always be new healthy validator group.